test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -180,6 +176,15 @@ public void initClient() throws IOException {
         assert hasXPack != null;
         assert nodeVersions != null;
     }
+
+    protected String getTestRestCluster() {
```

Comment:
I dare you to say "get test rest cluster" 10 times fast 😄

Reply:
lol, I'm not a native English speaker, I don't do fast.